src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+    public static void main(String[] args) {
+        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder()
+                .setModel("T25")
+                .setColor("White");
+
+        Plane planeYear = planeBuilder.setYearIssue(19990).build();
+        System.out.println(planeYear.toString());
+
+        Plane fastPlane = planeBuilder.setSpeed(3000).setYearIssue(2020).build();
+        System.out.println(fastPlane);
+
+        Plane breakPlane = planeBuilder.setSpeed(-1).build();
```

Comment:
does this line cause RuntimeException? if so, let's remove it from the main method

Reply:
done